FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -13,12 +13,36 @@ public final class RemoteFeedLoader: FeedLoader {
 		case invalidData
 	}
 
+	enum Status: Int {
+		case code_200 = 200
+	}
+
 	public init(url: URL, client: HTTPClient) {
 		self.url = url
 		self.client = client
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+
+			guard let self = self else { return }
+
+			switch result {
+			case .success((let data, let response)):
```

Comment:
The `let` keyword can be defined once outside the tuple resulting in a shorter expression:

```suggestion
			case let .success((data, response)):
```